Assets/Scripts/StuntSystem/Stunt.cs

```diff
@@ -0,0 +1,140 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public struct FrameKeyData
+{
+    public int frameCount;
+    public List<string> inputs;
+}
+
+public enum StuntType
+{
+    None,
+    Jump,
+    Spin,
+    Flip,
+    Grind
+}
+
+public class StuntDatabase
+{
+    public List<Stunt> stunts = new List<Stunt>();
+    public int frameDelayMax = 15;
+
+    public void PopulateStunts()
+    {
+        // Example of populating the stunt database
+        // If for efficiency, we can populate into a tree structure
+
+        // TODO: Add frame limit & priorities
+        // TODO: Single slides are broken atm. Absolutely needs priority
+
```

Comment:
This needs to be reworked in https://jira.internal.teamriders.dev/browse/RDS-97 and https://jira.internal.teamriders.dev/browse/RDS-104 to provide a more efficient way of developing stunts and storing them